photon-api/src/main/scala/com/linkedin/photon/ml/optimization/DistributedOptimizationProblem.scala

```diff
@@ -23,8 +23,9 @@ import com.linkedin.photon.ml.data.LabeledPoint
 import com.linkedin.photon.ml.function.{DistributedObjectiveFunction, L2Regularization, TwiceDiffFunction}
 import com.linkedin.photon.ml.model.Coefficients
 import com.linkedin.photon.ml.normalization.NormalizationContext
+import com.linkedin.photon.ml.optimization.game.GLMOptimizationConfiguration
 import com.linkedin.photon.ml.sampler.DownSampler
-import com.linkedin.photon.ml.supervised.model.{GeneralizedLinearModel, ModelTracker}
+import com.linkedin.photon.ml.supervised.model.{ModelTracker, GeneralizedLinearModel}
```

Comment:
This is absolutely crucial. Photon wouldn't even work without this change ;-) But should we not alphabetize on the lines as well as across lines?